<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LP3SET9" ignored="0" name="Test" tests="4" failures="1" timestamp="2023-03-02T17:02:08 IST" time="37.177" errors="0">
  <testcase name="addPaymentTest" time="1.024" classname="Org_InsuranceDomain_Clients.AddPaymentTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Org_InsuranceDomain_Clients.AddPaymentTest.addPaymentTest(AddPaymentTest.java:26)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- addPaymentTest -->
  <testcase name="updateClientTest" time="3.844" classname="Org_InsuranceDomain_Clients.UpdateClientTest"/>
  <testcase name="addNomineeTest" time="15.012" classname="Org_InsuranceDomain_Clients.AddNomineeTest"/>
  <testcase name="addClientTest" time="14.641" classname="Org_InsuranceDomain_Clients.AddClientTest"/>
</testsuite> <!-- Test -->
